pyproject.toml

```diff
@@ -94,11 +95,12 @@ docs = [
     "pydot",
     "pillow>=4.2.1",
     "reno>=3.4.0",
-    "qiskit-sphinx-theme~=1.14.0rc1",
+    "qiskit-sphinx-theme==1.16.1",
     "matplotlib>=3.4",
     "sphinx-reredirects",
     "sphinxemoji",
     "ipykernel",
+    "lxml_html_clean",
```

Comment:
What is this used for?

Reply:
Currently if you run `nox -edocs` with Python 3.13 it fails. I had to add this dependency to make it work.

It has to do with Python 3.13 resolving a newer lxml or something of sorts.